fast-sim-and-analysis/FccFastSimAnalysis.md

```diff
@@ -23,71 +23,64 @@ git clone https://github.com/HEP-FCC/FCCAnalyses.git
 cd FCCAnalyses
 ```
 
-This tutorial runs with fcc-edm so we need to checkout the corresponding branch
-
-
-```bash
-git checkout fccedm
-```
-
 and follow the compilation instructions [here](https://github.com/HEP-FCC/FCCAnalyses/#getting-started) to get started with **ONLY** the installation.
 Once the code has been compiled, we can now run the pre-selection on previously produced samples:
 
 ```bash
-python FCCeeAnalyses/ZH_Zmumu/dataframe/analysis.py PATH_TO_FILES_PART_I/p8_ee_ZH_ecm240.root
-python FCCeeAnalyses/ZH_Zmumu/dataframe/analysis.py PATH_TO_FILES_PART_I/p8_ee_ZZ_ecm240.root
-python FCCeeAnalyses/ZH_Zmumu/dataframe/analysis.py PATH_TO_FILES_PART_I/p8_ee_WW_ecm240.root
+python examples/FCCee/higgs/mH-recoil/mumu/analysis.py PATH_TO_FILES_PART_I/p8_ee_ZH_ecm240.root
```

Comment:
What is `PATH_TO_FILES_PART_I`?

Reply:
this is the path to the file produced in part 1 (with k4SimDelphes) the two are somehow independent